### PR TITLE
WFLY-4570 Add a systemd service

### PR DESCRIPTION
**Pull request by rarguello**
An example systemd service.

### CODE REVIEW

File: core-feature-pack/src/main/resources/content/bin/systemd/README [hunk 1]
```diff
@@ -0,0 +1,26 @@
+= How to configure WildFly as a systemd service
+
+== Create a wildfly user
+
+    # groupadd -r wildfly
+    # useradd -r -g wildfly -d /usr/share/wildfly -s /sbin/nologin wildfly
+
+== Install WildFly
+
+    # mkdir /usr/share/wildfly/
```
Comment (dmlloyd) [inline core-feature-pack/src/main/resources/content/bin/systemd/README]: Installation should probably be in /opt.  Data files should be (I guess) in /var/opt.  I would at least use http://www.linuxbase.org/betaspecs/fhs/fhs.html as a reference.

**Comment by dmlloyd** [comment]
OK but I think we need different default configurations for this then, because we don't want to put data files under /opt, they should be under /var, in particular the logging and data directories need to be separated.  So maybe this should continue to just be an example rather than something we expect people to typically use.

**Comment by rarguello** [comment]
This should be considered an example only, like the init.d scripts:
https://github.com/wildfly/wildfly-core/tree/master/core-feature-pack/src/main/resources/content/bin/init.d

**Comment by jamezp** [comment]
I'm placing a hold on this for now until we can determine how we want to do this or if we want to do this. I personally don't like any of the extra scripts in the code base.

**Comment by ctomc** [comment]
@dmlloyd @jamezp i think we should merge this, as everything is moving to systemd and even having a starting point is a good thing.

**Comment by ctomc** [comment]
this is ok to test

**Comment by ctomc** [comment]
I squashed your commits in this PR and included them in #1132

**Comment by rarguello** [comment]
Thanks

**Pull request closed without merging**